src/Dhadgar.Scope/src/components/visualizations/DependencyGraph.tsx

```
@@ -1,445 +1,445 @@
+        />
+      </div>
+
+      {/* Tablet drawer */}
+      {drawerOpen && (
+        <>
+          <div
+            role="presentation"
+            className="fixed inset-0 z-50 hidden bg-black/60 md:block lg:hidden"
+            onClick={() => setDrawerOpen(false)}
+            onKeyDown={(e) => e.key === "Escape" && setDrawerOpen(false)}
+          />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)